be/src/vec/columns/column_map.cpp

```diff
@@ -450,7 +450,8 @@ void ColumnMap::reserve(size_t n) {
 }
 
 void ColumnMap::resize(size_t n) {
-    get_offsets().resize(n);
+    auto last_off = get_offsets().back();
+    get_offsets().resize_fill(n, last_off);
     keys_column->resize(n);
```

Comment:
should not call resize on keys and values column

Reply:
done!